src/components/pager/js/Pager.js

```diff
@@ -295,8 +295,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     });
 
     fluid.pager.previousNext.update = function (that, disabledStyle, newModel) {
-        that.previous.toggleClass(disabledStyle, newModel.pageIndex === 0);
-        that.next.toggleClass(disabledStyle, newModel.pageIndex === newModel.pageCount - 1);
+        var isFirstPage = (newModel.pageIndex === 0);
```

Comment:
The handling for the previous and next links are pretty much same with the difference on the calculation of the flag (isFirstPage & isLastPage), can you create a generic function with the flag being the pass-in parameter to share this handling? Thanks.